### PR TITLE
CLIO-36783: Added 'header' style to thTabset

### PR DESCRIPTION
**Pull request by craigcarlyle**

### CODE REVIEW

File: themis_components/thTabset/examples/5 - Header style/html.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<html ng-app="thDemo">
+  <body>
+
+    <th-tabset type="header">
+      <th-tab name="Tab One">
+        <h4>Tab One</h4>
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
```
Comment (JefCurtis) [inline themis_components/thTabset/examples/5 - Header style/html.html]: Too bloody long...
giphy

File: themis_components/thTabset/index.scss [hunk 2]
```diff
@@ -22,17 +22,28 @@
...
-      font-size: 15px;
 
       &.active {
         font-weight: bold;
       }
     }
   }
+  &.header {
+    a {
+      margin-right: 24px;
```
Comment (JefCurtis) [inline themis_components/thTabset/index.scss]: Can these be grouped by type?

File: themis_components/thTabset/examples/5 - Header style/html.html (hunk 1 above)
Comment (craigcarlyle) [inline themis_components/thTabset/examples/5 - Header style/html.html]: Not according to our test suite

File: themis_components/thTabset/index.scss (hunk 2 above)
Comment (JefCurtis) [inline themis_components/thTabset/index.scss]: As per @seanhealy something like:
```css
/* Display & Box Model */
margin-right: 24px;
padding-bottom: 12px;

/* Text */
font-size: 18px;
```

File: themis_components/thTabset/examples/5 - Header style/html.html (hunk 1 above)
Comment (JefCurtis) [inline themis_components/thTabset/examples/5 - Header style/html.html]: It's NOT checking HTML files but if it requires scrolling horizontally in github it is def too long.

File: themis_components/thTabset/index.scss (hunk 2 above)
Comment (craigcarlyle) [inline themis_components/thTabset/index.scss]: Addressed.

File: themis_components/thTabset/readme.md [hunk 3]
```diff
@@ -41,6 +42,29 @@ Currently `sub-header` is the only option.
 </th-tabset>
 ```
 
+You can also include `thButton` and `thDropdown` elements by placing them in an `th-tab-action-bar` element.
```
Comment (JefCurtis) [inline themis_components/thTabset/readme.md]: Anything is transcluded here. Should we just say so and use the obvious examples of button / dropdowns?

File: themis_components/thTabset/examples/6 - Alternate styles/html.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<html ng-app="thDemo">
```
Comment (JefCurtis) [inline themis_components/thTabset/examples/6 - Alternate styles/html.html]: Thought. Could you wrap these examples similar to the `withMessage` component examples. I think it makes it much clearer.
screenshot 2016-07-22 09 44 16

Comment (craigcarlyle) [inline themis_components/thTabset/examples/6 - Alternate styles/html.html]: Great suggestion. Will do.

File: themis_components/thTabset/readme.md (hunk 3 above)
Comment (craigcarlyle) [inline themis_components/thTabset/readme.md]: Yup.

File: themis_components/thTabset/readme.md [hunk 5]
```diff
@@ -41,6 +42,29 @@ Currently `sub-header` is the only option.
 </th-tabset>
 ```
 
+You can also include transclude elements (such as `thButton` and `thDropdown`) by placing them in an `th-tab-action-bar` element:
+
+### Markup with Buttons
+
+```
```
Comment (JefCurtis) [inline themis_components/thTabset/readme.md]: Adding a language type to your code block will add syntax highlighting! 💯 
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

```
<th-tabset type="header"></th-tabset>
```
VS
```html
<th-tabset type="header"></th-tabset>
```

**Pull request merged**